ext/mysqli/mysqli.stub.php

```diff
@@ -256,10 +256,10 @@ public function set_opt(int $option, $value) {}
      */
     public function ssl_set(
         string $key,
-        string $cert,
+        string $certificate,
         string $certificate_authority,
         string $certificate_authority_path,
-        string $cipher
+        string $cipher_algorithms
```

Comment:
```suggestion
        string $cipher_algos
```
Per recent change in openssl.